_studio/mfx_lib/decode/vp9/src/mfx_vp9_dec_decode_hw.cpp

```diff
@@ -134,17 +134,36 @@ mfxStatus VideoDECODEVP9_HW::CleanRefList()
 class FrameStorage
 {
 private:
+    void CheckUmcStatus(UMC::Status status) {
```

Comment:
The method doesn't use any _FrameStorage_ state, why it isn't free function?

Reply:
Done